src/DataSets/context.actions.js

```diff
@@ -57,24 +60,35 @@ const canUpdate = (d2, datasets) => {
     );
 }
 
+function logNRun(actionName, f) {
+    // Return a function that, when called with a dataset, logs the
+    // actionName and some info related to the dataset, and then calls
+    // f(dataset)
+    return (dataset) => {
+        //log(actionName, 'started', dataset);
+        // if uncommented, will log the fact that we tried to start the action.
+        f(dataset);
+    }
+}
+
```

Comment:
But do we need logs on action starts? it's to keep the function as clean as possible, we try to minimize commented code.

Reply:
I agree, this came from a conversation between @adrianq and me, as we were not completely sure. I'd vote for removing this part.

Reply:
Then go ahead and just delete the comment! :)

Reply:
Great, done :)